### PR TITLE
TDP-4829 cnam docs

### PR DESCRIPTION
**Pull request by muesono-daitan**
The new page can be seen here:
http://bw-dev-tdp-4829.s3-website-us-east-1.amazonaws.com/voice/guides/cnamService.html

## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

Customers frequently ask how to look up number info (CNAM) because Catapult offered it. Our docs should point to those docs somewhere.

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4829.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4829.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4829.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4829.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4829.s3-website-us-east-1.amazonaws.com/index.html

**Review by lpinheiro-daitan — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4829.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4829.s3-website-us-east-1.amazonaws.com/index.html

File: voice/guides/cnamService.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+`https://cnam.dashcs.com/?companyId=<companyId>&password=<pass>&number=<phone number>`
+
+Note: Please ensure that the password is URL-encoded.
+
+## Query parameters 
+| Query Type                | Required or Optional | Type                     | Value/Notes |
+|---------------------------|----------------------|--------------------------|-------------|
+| CompanyId                 | Required             | Integer                  | Assigned during implementation/onboarding. |
+| Password                  | Optional             | String                   | Assigned during implementation/onboarding. |
+| Number                    | Required             | 11-digit TN              | (1+10 digit TN) that's used to query CNAM information. |
```
Comment (scapps-bw) [inline voice/guides/cnamService.md]: should still be specific that Number and DNIS are `string` s

File: voice/guides/cnamService.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+With this service, the CNAM database is queried using a simple HTTPS GET Request.
+Once the request is made, the calling name is returned to the customer application or dial plan.
+The CNAM data is managed by the phone company who provides the phone numbers to its customers.
+Bandwidth can not guarantee the validity of the information returned by the managing phone company.
+
+To utilize the CNAM lookup API, you will need to be setup with an account by your Account Manager.
+Please open a [support ticket](https://support.bandwidth.com/hc/en-us/requests/new) to enable this feature for your account.
+
+## CNAM setup instructions via HTTPS
+The simplest method of accessing the CNAM service is via an HTTPS GET request.
```
Comment (scapps-bw) [inline voice/guides/cnamService.md:14]: are there other methods for accessing this specific service?

File: voice/guides/cnamService.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Query format
+`https://cnam.dashcs.com/?companyId=<companyId>&password=<pass>&number=<phone number>`
+
+Note: Please ensure that the password is URL-encoded.
+
+## Query parameters 
+| Query Type                | Required or Optional | Type                     | Value/Notes |
+|---------------------------|----------------------|--------------------------|-------------|
+| CompanyId                 | Required             | Integer                  | Assigned during implementation/onboarding. |
+| Password                  | Optional             | String                   | Assigned during implementation/onboarding. |
```
Comment (scapps-bw) [inline voice/guides/cnamService.md]: how can the password be optional?

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4829.s3-website-us-east-1.amazonaws.com/index.html

File: voice/guides/cnamService.md (hunk 1 above)
Comment (muesono-daitan) [reply voice/guides/cnamService.md]: Changed the way it was written.

File: voice/guides/cnamService.md (hunk 2 above)
Comment (muesono-daitan) [reply voice/guides/cnamService.md:14]: Changed the way it was written. Maybe it makes a little more sense now?
There are other two ways (non-api) linked at the end of the doc.

File: voice/guides/cnamService.md (hunk 3 above)
Comment (muesono-daitan) [reply voice/guides/cnamService.md]: Fixed.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4829.s3-website-us-east-1.amazonaws.com/index.html

**Review by scapps-bw — APPROVED** [review]

**Pull request merged**